components/centraldashboard/public/components/main-page.js

```diff
@@ -245,11 +293,21 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
      * @param {Event} responseEvent AJAX-response
      */
     _onEnvInfoResponse(responseEvent) {
-        const {platform, user, namespaces} = responseEvent.detail.response;
-        this.user = user;
-        this.namespaces = namespaces;
+        const {platform, user,
+            namespaces, isClusterAdmin} = responseEvent.detail.response;
+        Object.assign(this, {user, isClusterAdmin});
+        this.namespaces = namespaces.map((n) => ({
```

Comment:
Can't because the iron-ajax is not in that component, and the registrationFlow state (in main-app) gets unset when namespace count is 0 (which is what allows the flow page to close and show the central-dashboard.

Reply:
I don't mean the ajax request itself but I just meant passing the response that's collected here directly to the child component(s) that deal with the namespace (BindingEntries) data. Just make sure you add/fix the unit tests that cover this.